src/service/standard-actions-impl.js

```diff
@@ -101,10 +112,14 @@ export class StandardActions {
    * @param {number=} opt_actionIndex
    * @param {!Array<!./action-impl.ActionInfoDef>=} opt_actionInfos
    * @return {?Promise}
-   * @throws {Error} If action is not recognized.
+   * @throws {Error} If action is not recognized or is not whitelisted.
    */
   handleAmpTarget(invocation, opt_actionIndex, opt_actionInfos) {
     const method = invocation.method;
+    if (this.ampActionWhitelist_ &&
+      this.ampActionWhitelist_.indexOf(method) == -1) {
```

Comment:
`!this.ampActionWhitelist_.includes(method)`. `Array#include` is polyfilled here:

https://github.com/ampproject/amphtml/blob/master/src/polyfills/array-includes.js

Reply:
done